integration-tests/scripts/delete-single-branch.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
Seems like you should have a separated file with some common functions. It is repeating too much.

Reply:
I have refactored the scripts to include functions included from a library.

Reply:
I was talking specifically about the github part. You wrote the same stuff several times at the start of each file that needs GITHUB_TOKEN set. I said that this should be part of you library function - some `load_github_env_vars()` or something. But I am not blocking your PR because of this.  It is just something to keep in mind. As you said, this should be temporary until the proper e2e is wrote.